index.js

```diff
@@ -9,7 +9,23 @@ app.use(express.urlencoded({extended : false}));
 app.use(authentication.initialize());
 app.use(apiRouter);
 app.use('/api/v1/api-docs', swaggerUi.serve, swaggerUi.setup(docs));
-const PORT = process.env.PORT || 3000
+const PORT = process.env.PORT || 3000;
+const docsUrl = 'https://way-farer-app-rest.herokuapp.com/api/v1/api-docs/';
+// internal server error implementation to use when database will be integrated
+/*app.use((err, req, res, next) => {
```

Comment:
@WinnersProx , please remove redundant lines of code. such as these commented lines

Reply:
Thanks, I get it dear